src/main/java/gov/cms/madie/models/validators/ScoringPopulationValidator.java

```diff
@@ -50,33 +51,55 @@ public boolean isValid(
     final String popBasis = testCaseGroupPopulation.getPopulationBasis();
 
     boolean allValuesMatchPopulationBasis = popBasis == null || populationValues.stream().allMatch(popVal -> {
-      if (popVal == null || popVal.getExpected() == null) {
-        return true;
-      } else if ("Boolean".equals(popBasis)) {
-        if(popVal.getName().equals(PopulationType.MEASURE_OBSERVATION) && !popVal.getExpected().equals(false)){
-          return popVal.getExpected() instanceof String;
-        }else {
-          return popVal.getExpected() instanceof Boolean;
-        }
-      } else if (popVal.getExpected().toString().isEmpty()) {
+      
+      if (popVal == null || popVal.getExpected() == null || popVal.getExpected().toString().isEmpty()) {
         return true;
       } else {
-        try {
-          if(!popVal.getName().equals(PopulationType.MEASURE_OBSERVATION)){
-            Integer.parseInt(popVal.getExpected().toString());
-            return true;
+        boolean isObv = isObservation(popVal.getName());
+        boolean getExp = popVal.getExpected().equals(false);
+        //isObservation needs to be a number, regardless of populationBasis
+        //!isObservation needs to match popBasis type (ie., Boolean or Number)
+
+        if(isObservation(popVal.getName())) {
```

Comment:
did you mean to use your `isObv` variable from a few lines up

Reply:
Nope.. just forgot to clean that up; fixed now